tests/analytical_comparisons/analytical.py

```diff
@@ -72,7 +73,8 @@
 
 
 def get_case(cases, config):
-    config = config.split("/")
+    print(config)
```

Comment:
```suggestion
```

Drop?

Reply:
Yes please. Looks like I left some debugging in...